src/app/icd/ICDManager.cpp

```diff
@@ -32,8 +34,13 @@ using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::IcdManagement;
 
-void ICDManager::ICDManager::Init()
+void ICDManager::ICDManager::Init(PersistentStorageDelegate * pStorage, FabricTable * pFabricTable)
 {
+    VerifyOrDie(pStorage != nullptr);
+    VerifyOrDie(pFabricTable != nullptr);
+    mpStorage     = pStorage;
```

Comment:
Not a huge fan of the "p" prefixes all over here, but either way, I guess...

Reply:
Removed the `p`. It was to represent the pointer type, but I am not attached to it.